.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,73 @@
+name: Deploy and Configure Morio
+
+on:
+  workflow_dispatch:  # This allows the workflow to be triggered manually
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '20'  # Replace with the Node.js version you need
+
+      - name: Install AWS CLI
+        run: |
+          curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+          unzip awscliv2.zip
+          sudo ./aws/install
+          aws --version
+
+      - name: Configure AWS CLI
+        run: |
+          aws configure set aws_access_key_id ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws configure set aws_secret_access_key ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws configure set region ${{ secrets.AWS_REGION }}  # You can specify the region here, e.g., 'us-west-2'
+          aws configure set output json  # Set the output format to JSON
+
+      - name: Install Python 3 and pip
```

Comment:
GitHub Actions runners also come with Pything pre-installed 😇

Reply:
I've removed it also. :)